packages/editor-ui/src/stores/workflows.store.ts

```diff
@@ -1338,12 +1339,25 @@ export const useWorkflowsStore = defineStore(STORES.WORKFLOWS, {
 
 		async runWorkflow(startRunData: IStartRunData): Promise<IExecutionPushResponse> {
 			const rootStore = useRootStore();
-			return await makeRestApiRequest(
-				rootStore.getRestApiContext,
-				'POST',
-				'/workflows/run',
-				startRunData as unknown as IDataObject,
-			);
+			try {
+				return await makeRestApiRequest(
+					rootStore.getRestApiContext,
+					'POST',
+					'/workflows/run',
+					startRunData as unknown as IDataObject,
+				);
+			} catch (error) {
+				if (error.response?.status === 413) {
+					throw new ResponseError(
+						'Please execute the whole workflow, rather than just the node. (Existing execution data is too large.)',
+						{
```

Comment:
Can you use a translation string for this instead?

You can import it directly from the plugins to use it inside of stores:

```
import { i18n } from '@/plugins/i18n';
```

I'll approve since this is not a blocker, but would be nice to have.

Reply:
Ah good idea, changed to use i18n and tested. Could you please reapprove?